modules/plugin/http-commons/src/test/java/org/geotools/http/commons/MultithreadedHttpClientTest.java

```diff
@@ -81,6 +81,25 @@ public void testGetWithoutNonProxyHost() throws MalformedURLException, IOExcepti
         wireMockProxyRule.verify(getRequestedFor(urlEqualTo("/fred")));
     }
 
+    @Test
+    public void testWithBasicAuthProvided() throws Exception {
```

Comment:
is this basically the same test as `testBasicHeaderGET()` down below?

Reply:
almost, mine does not expect a 401 response before sending the basic authentication, which is the purpose of my PR. The 401 will be fully managed by the apache http client anyway.